app/authenticate.py

```diff
@@ -0,0 +1,34 @@
+from flask_httpauth import HTTPTokenAuth, HTTPBasicAuth, MultiAuth
+from flask import jsonify, g
+
+from app.models import User
+
+token_auth = HTTPTokenAuth("Bearer")
+passw_auth = HTTPBasicAuth()
```

Comment:
Consider using the full name for better understanding of your code base. Instead of `pass_w` use the full `password_authentication`. Its more friendly.